### PR TITLE
Academy-related List Component CSS Update

### PR DESCRIPTION
**Pull request by sghoweri**
## Jira
http://vjira2:8080/browse/BDS-1957

## Summary
Small CSS update to List based on Mission Landing template build

## Details
- moved border to right side
- updated the List's border color to now use the `bolt-theme` function + opacity.

## How to test
- Review changes
- Double check List demos in PL

### CODE REVIEW

File: packages/components/bolt-list/src/_list-item.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 // Variables
 $bolt-list-item-border-width: $bolt-border-width;
-$bolt-list-item-border-color: $bolt-border-color;
+$bolt-list-item-border-color: bolt-theme(border, 0.3);
```
Comment (mikemai2awesome) [inline packages/components/bolt-list/src/_list-item.scss]: @sghoweri @danielamorse 

This is different than border colors used in the following components, what's the reason for breaking the consistency?

## Using `bolt-theme(border, 0.2)`:

* action blocks
* card
* ui list

List was using the `$bolt-border-color: rgba(bolt-color(gray), 0.2)` global variable, which is the same as `bolt-theme(border, 0.2)`. Can we keep all borders consistent? If not, what's the plan for creating variations of border colors and styles?

Comment (sghoweri) [reply packages/components/bolt-list/src/_list-item.scss]: @mikemai2awesome could you change this to 0.2 please?

Comment (mikemai2awesome) [reply packages/components/bolt-list/src/_list-item.scss]: Updated.

**Comment by sghoweri** [comment]
> @sghoweri CSS fix is pushed.

Thanks @mikemai2awesome!

**Comment by sghoweri** [comment]
PR was released with v2.15.0

**Pull request merged**